lib/demos/material/sliders_demo.dart

```diff
@@ -321,32 +319,35 @@ class _CustomRangeThumbShape extends RangeSliderThumbShape {
     );
 
     final size = _thumbSize * sizeTween.evaluate(enableAnimation);
-    Path thumbPath;
+    late Path thumbPath;
     switch (textDirection) {
```

Comment:
not sure if this is the right approach for this function , but made the appropriate one default